apps/trading/client-pages/markets/top-market-list.tsx

```diff
@@ -0,0 +1,33 @@
+import { EmblemByMarket } from '@vegaprotocol/emblem';
+import type { MarketMaybeWithCandles } from '@vegaprotocol/markets';
+import { useChainId } from '@vegaprotocol/wallet-react';
+import { Link } from 'react-router-dom';
+import { priceValueFormatter, priceChangeRenderer } from './use-column-defs';
+
+export const TopMarketList = ({
+  markets,
+}: {
+  markets?: MarketMaybeWithCandles[];
+}) => {
+  const { chainId } = useChainId();
+  return (
+    <div className="grid auto-rows-min grid-cols-6 gap-3 text-sm">
+      {markets?.map((market) => {
+        return (
+          <>
+            <span className="col-span-3">
+              <Link to={`/markets/${market.id}`}>
+                <span className="pr-1">
+                  <EmblemByMarket market={market.id} vegaChain={chainId} />
```

Comment:
**chore (blocking)** : `chainId` is defaulting to `mainnet` here, presumably because `useChainId` is not populated when the wallet is not connected. Or something like that. Either way, the chain ID needs to be set properly for all users, which it isn't - this is why the logos in the deploy previews are black. In explorer, this is set in `.env` (to stagnet) and then more explicitly in every other `.env.*`.

Reply:
I have updated all `.env.*`